src/igvfd/schemas/biosample.json

```diff
@@ -254,18 +254,18 @@
                 "decline"
             ]
         },
-        "biosample_ontology": {
-            "title": "Biosample ontology term",
-            "description": "Ontology term of the biosample.",
+        "biosample_term": {
+            "title": "Biosample term",
+            "description": "Term of the biosample.",
             "type": "string",
-            "linkTo": "SampleOntologyTerm"
+            "linkTo": "SampleTerm"
         },
-        "disease_ontology": {
-            "title": "Disease ontology term",
-            "description": "Ontology term of the disease associated with the biosample.",
+        "disease_term": {
+            "title": "Disease term",
+            "description": "Term of the disease associated with the biosample.",
```

Comment:
include indication that it is ontological term

Reply:
done